spec/avram/slugify_spec.cr

```diff
@@ -123,6 +115,40 @@ describe Avram::Slugify do
       op.slug.value.to_s.should start_with("the-boss-") # Has UUID appended
     end
   end
+
+  describe ".generate" do
+    it "skips blank slug candidates" do
+      slug = Avram::Slugify.generate(["", "Software Developer"])
+
+      slug.should eq("software-developer")
+    end
+
+    describe "with a single slug candidate" do
+      it "sets slug from a single attribute" do
+        op = build_op(title: "Software Developer")
+        slug = Avram::Slugify.generate(op.title)
+
+        slug.should eq("software-developer")
+      end
+
+      it "sets slug from a single string" do
+        slug = Avram::Slugify.generate("Software Developer")
+
+        slug.should eq("software-developer")
+      end
+    end
+
+    describe "with an array of slug candidates" do
+      it "sets when using multiple attributes" do
+        op = build_op(title: "How Do Magnets Work?", sub_heading: "And Why?")
+
+        slugify(op.slug, [[op.title, op.sub_heading]])
```

Comment:
What's this line do? Is it proving a compile time check?

Reply:
I put it there just to make sure you were awake :stuck_out_tongue:  actually, it's just a copy/paste error. I need to remove it :sweat_smile: